lib/preneeds/middleware/response/clean_response.rb

```diff
@@ -4,7 +4,7 @@ module Middleware
     module Response
       class CleanResponse < Faraday::Response::Middleware
```

Comment:
agree, their errors are terrible though -- it comes back as an XML stacktrace, no concept of minor codes, so not exactly sure how to tackle that at this point.